rmw_opensplice_cpp/src/rmw_event.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright 2014-2019 Open Source Robotics Foundation, Inc.
```

Comment:
```suggestion
// Copyright 2019 Open Source Robotics Foundation, Inc.
```

Reply:
Nitpick: If this file hasn't been in development since 2014 then the date should start at 2019